Tribler/Core/CacheDB/Notifier.py

```diff
@@ -69,6 +68,7 @@ def remove_observers(self):
             self.observers = []
 
     def notify(self, subject, changeType, obj_id, *args):
+        from Tribler.Core.Utilities.twisted_thread import callInThreadPool
```

Comment:
hm, I didn't think of that... wait a sec, I'll move the utility functions out of the twisted_thread module instead.